tests/helpers.py

```diff
@@ -1111,6 +1111,44 @@ def measure_pvc_deletion_time(interface, pv_name):
     return total.total_seconds()
 
 
+def pod_start_time(pod_obj):
+    """
+
+    Function to measure time taken for container(s) to get into running state
```

Comment:
Please leave a little bit more comments how function works, i did not get the logic straightaway when I read the code.

Reply:
Added some more description.